tests/acceptance/17_packages/10_new/unsafe/packagesmatching.cf

```diff
@@ -0,0 +1,55 @@
+# Test that updates list cache is not updated too often.
+
+body common control
+{
+    inputs => { "../../../default.cf.sub" };
+    bundlesequence => { default($(this.promise_filename)) };
+
+    package_inventory => { test_module };
+    #package_module => "test_module";
+}
+
+bundle agent init
+{
+  meta:
+    "test_skip_needs_work" string => "!linux";
+
+  files:
+    test_pass_1::
+      "$(sys.workdir)/modules/packages/test_module"
+        copy_from => local_cp("$(this.promise_dirname)/test_module"),
+        perms => m("0755");
+}
+
+bundle agent test
+{
+  commands:
+    test_pass_1::
```

Comment:
The test needs to reside in a `timed/` directory for this to work. So you'll need two levels of directories: `unsafe/timed/`.